packages/ui/primitives/document-flow/add-signers.tsx

```diff
@@ -174,6 +164,20 @@ export const AddSignersFormPartial = ({
     removeSigner(index);
   };
 
+  const onAddSelfSigner = () => {
+    const lastSignerIndex = signers.length - 1;
+    if(!signers[lastSignerIndex].name || !signers[lastSignerIndex].email){
```

Comment:
Your spacing is messed up, I think it might be your extensions? No space after `if` and no space before `{`

This if statement is incorrect, if the last signer has an email but no name it gets replaced?

Reply:
This isn't resolved, did you read the whole message?

Look below, if a previous SAVED recipient has no name, adding yourself removes them. 

https://github.com/documenso/documenso/assets/20962767/731c3e70-28f3-421d-bc75-680ff47502f7